src/elements/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { useNavigation } from 'react-navigation-hooks';
+import { View, Text, SafeAreaView } from 'react-native';
```

Comment:
I noticed in your write up that you recommended SafeAreaView - I had to research this and totally agree with you, seems like a great idea!  But, I don't see where you used it in your code?

Reply:
So now I'm playing around with it in your branch locally, and I see what you mean!  Use SafeAreaView in our screens, with {flex: 1} style.  We should decide what background color - light gray? Great recommendation, Tao!!
![image](https://user-images.githubusercontent.com/44376276/80643342-c65e2f00-8a1c-11ea-9927-70e07716955f.png)

Reply:
I'm curious, if the SafeAreaView will have to be used as a wrapper on ***all screens*** and be given a flex: 1, why not add the SafeAreaView and flex style prop to the NavBar component?

Reply:
Totally agree, and the reason I did not use SafeAreaView since it does not work if it's only applied to NavBar. As we discussed on slack, I would create a PR that use a SafeAreaView to wrap all screen at app.tsx, thanks for the suggestion!